GoogleForCreators/web-stories-wp
# Build tool: work on copy, not source directory

## Summary

Updates the build tool so that any file changes (e.g. changing the CDN URL) are only applied to the actual build (i.e. `build/web-stories/include/namespace.php`) and not the source file.

## To-do

N/A

## User-facing changes

N/A

## Testing Instructions

1. Run `npm run workflow:build-plugin -- --composer --cdn --zip`.
1. Verify that source files stay unchanged (i.e. no changes to be committed).
1. Verify that `WEBSTORIES_ASSETS_URL` in `build/web-stories/web-stories.php` has been updated.
1. Install ZIP file on website and verify that the plugin still works.
